protos/flyteidl/admin/schedule.proto

```diff
@@ -16,6 +16,12 @@ message FixedRate {
     FixedRateUnit unit = 2;
 }
 
+// Supported only by some schedulers for example Spotify/Styx. Your platform may not have this enabled.
+message CronScheduleWithOffset {
+    string schedule = 1;
```

Comment:
is the schedule here actually the expression?
Also, this is pretty good, i think eventually we should just deprecate cron_expression for CronScheduleWithOffset (just offset will be 0 by default and not supported on most platforms)

Reply:
It can be a cron expression or alias such as hourly, daily, weekly, monthly etc. So we were not so sure what name could be more reasonable.

Reply:
I think this PR https://github.com/lyft/flytekit/pull/188 gives more details.